condensables.md

```diff
@@ -7,7 +7,7 @@ title: Condensable items
 
 # Condensable items
 
-The following list contains all the valid condensable items that can be used to generate Artron Energy. The number represents the relative energy value (i.e. Item value:1 != Energy value:1).
+The following list contains all the valid condensable items that can be used to generate Artron Energy. The number represents the relative energy value, for example, 1 Apple will give you 10 Artron Energy.
```

Comment:
Actually an Apple will only give you [5 Artron energy](https://github.com/eccentricdevotion/TARDIS/blob/v4.0/src/main/java/me/eccentric_nz/TARDIS/artron/TARDISCondenserListener.java#L202)

Reply:
Really? Why does the actual list say 10? I just checked and yeah it's 5

Reply:
Oh I get it. The relative value is twice the Artron energy given? Where is the relative value used?

Reply:
It's never used, I'm just too lazy to change all the current values to half what they are now

Reply:
I see a PR incoming...

Reply:
Hm. Wouldn't that break existing configs that account for the x2?

Reply:
Go for it, but as far as code optimisation goes there'd be only an infinitesimal benefit.

Reply:
Only the values from the condensables list are halved all other values are 1:1 eg artron.yml